components/filters.tsx

```
@@ -123,47 +165,189 @@ export function CategoriesList({
+                onChange={(e) => setSearchTerm(e.target.value)}
+                className="pl-10 pr-12"
+                classNames={{
+                  input: "bg-gray-50 dark:bg-gray-900/50 border-gray-300 dark:border-gray-600/50 text-gray-900 dark:text-white placeholder-gray-500 dark:placeholder-gray-400",
+                  inputWrapper: "bg-gray-50 dark:bg-gray-900/50 border-gray-300 dark:border-gray-600/50 hover:border-gray-400 dark:hover:border-gray-500 focus-within:border-blue-500 dark:focus-within:border-blue-400"
+                }}
+              />
+              {searchTerm && (
+                <button
+                  onClick={() => setSearchTerm("")}
+                  className="absolute inset-y-0 right-0 pr-3 flex items-center text-gray-400 dark:text-gray-500 hover:text-gray-600 dark:hover:text-gray-300 transition-colors"
+                >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)